searx/searx
# [fix] update buildenv

## What does this PR do?

CONTACT_URL is unset in Makefile, but searx/brand.py and
utils/brand.env are not updated.

This commit fixes this issue.

## Why is this change important?

Without this commit the contact information at the bottom of the page contains a misleading information (even if it is obvious that contact@example.com is wrong).

## How to test this PR locally?

* `make run`
* check at the bottom at the page that the contact information is not displayed.

## Author's checklist



## Related issues
